projects/packages/my-jetpack/_inc/components/onboarding-screen/connection-form.module.scss

```diff
@@ -0,0 +1,67 @@
+.connection-form {
+	display: flex;
+	flex-direction: column;
+	gap: 16px;
+	width: 100%;
+	max-width: 450px;
+}
+
+.title {
+	font-size: 32px;
+	font-weight: 500;
+}
+
+.description {
+	font-size: 16px;
+	font-weight: 400;
+	color: var(--jp-gray-100, #101517);
+}
+
+.social-button {
+    display: flex;
+    justify-content: center;
+	align-items: center;
+    align-self: stretch;
+    margin-bottom: 0;
+    padding: 14px 16px;
+	width: 100%;
+	border: 1px solid #CCC;
```

Comment:
Let's write hex codes either lower or upper case. I think in the other instances we've decided on the lower one, so here it would be #ccc